Sources/Danger/GitLabDSL.swift

```diff
@@ -106,14 +106,14 @@ extension GitLab {
             }
 
             public enum CodingKeys: String, CodingKey {
-                case id // swiftlint:disable:this identifier_name
+                case id 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 109;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)